automation/jleechanorg_pr_automation/openai_automation/codex_github_mentions.py

```
@@ -38,34 +40,16 @@
     async_playwright,
 )
 
+from ..logging_utils import setup_logging as _setup_logging
```

Ruff (0.14.11)

Prefer absolute imports over relative imports from parent modules

Replace relative imports from parent modules with absolute imports

(TID252)